README.md

```
@@ -157,13 +157,23 @@ leverage tofu init              # Initialize the layer
 leverage tofu apply             # Apply changes
 leverage tofu destroy           # Destroy infrastructure
-leverage tofu fmt               # Format code
+leverage tofu format            # Format code (always recursive)
+leverage tofu format -check     # Check formatting without rewriting
 leverage tofu validate          # Validate configuration
-leverage tofu state list        # List resources in state
-leverage tofu state show <res>  # Show a specific resource in state
 ```
 
 > `leverage tf` is a shorthand alias for `leverage tofu`. Both run OpenTofu.
 
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)